packages/components/confirmation-prompt/stories/confirmation-prompt.happo.js

```diff
@@ -0,0 +1,58 @@
+import { html } from 'lit-html';
+import '@tradeshift/elements';
+import '@tradeshift/elements.modal';
+import '@tradeshift/elements.button';
+import '@tradeshift/elements.text-field';
+import '@tradeshift/elements.confirmation-prompt';
+
+export default {
+	title: 'ts-confirmation-prompt'
+};
+
+export const ConfirmationPrompt = () => {
+	return html`
+		<div style="width: 100%; height: 800px;"></div>
```

Comment:
I think the `height` should be 1200px because we specify viewport 1200x1200 in Happo settings:
https://github.com/Tradeshift/elements/blob/master/.happo.js#L10

Right now confirmation modals looks truncated by height, because I cannot see buttons on the bottom.